console.go

```diff
@@ -65,19 +65,17 @@ type WinSize struct {
 }
 
 // Current returns the current process' console
-func Current() (c Console) {
-	var err error
+func Current() (c Console, err error) {
```

Comment:
Instead of renaming this, can we add an additional constructor that returns the error?  Something like:

```go
func GetCurrent() (Console, error)
```

Reply:
Then we shouldn't have to worry about the breaking change.